external/fv3fit/fv3fit/pytorch/training_loop.py

```diff
@@ -52,14 +52,17 @@ def fit_loop(
             .batch(self.samples_per_batch)
         )
         train_data = tfds.as_numpy(train_data)
-        validation_data = validation_data.unbatch()
-        n_validation = sequence_size(validation_data)
-        validation_state = (
-            torch.as_tensor(next(iter(validation_data.batch(n_validation))).numpy())
-            .float()
-            .to(DEVICE)
-        )
-        min_val_loss = np.inf
+        if validation_data is not None:
```

Comment:
This was not previously tested without validation data (still isn't, but it's not within the scope of this PR).